crates/sock-prot/src/protector/windows.rs

```diff
@@ -1,25 +1,21 @@
 use libc::setsockopt;
-use parking_lot::Mutex;
```

Comment:
nit: I would keep using the parking_lot

Reply:
I don't see a reason why. It adds to a dependency that I deliberately tried to keep to a minimum

Reply:
The thing is that this is inherently fallible and `parking_lot` simply ignores the error.
If the poisoning happens, then there's no way to handle this IMHO https://doc.rust-lang.org/nomicon/poisoning.html, and ignoring the problem sometimes makes sense (what `parking_lot` is doing) in the case of primitives but not in the case of a complex structure being protected like `Vec`.
And the additional thing is that adds to the dependencies bag.

Reply:
In reality, in our case, the mutexes can become poisoned in case of panic and that's already bad enough. It probably can happen only in the case of OOM. I don't think there's a need to optimize for such a case